### DISCUSSION TITLE
请问最高支持什么版本的Laravel？

### DISCUSSION THREAD

**Discussion by fearlessxjdx** [ZsgsDesign/NOJ#647] [Q&A] [answered]
我试了试Laravel8，发现create-admin的时候总是报数据库错误，不知道是不是和版本有关系

**Comment by ZsgsDesign** [comment, marked as answer]
NOJ锁死的6呀，如果希望部署请按照[安装指南](https://njuptaaa.github.io/docs/#/noj/guide/deploy)进行。

**Reply by fearlessxjdx** [reply]
好的谢谢，可能是我安装说明看的不够仔细

**Reply by ZsgsDesign** [reply]
按道理不会遇到Laravel 8的……NOJ会在兼容以后升级新版Laravel，主要是使用的一些包没升级到Laravel 7和以上，不好升级，安装时需要使用`composer install`正确安装依赖包版本。
